src/_scoreboard.scss

```diff
@@ -0,0 +1,23 @@
+.app {
+	.scoreboard {
+		position: relative;
+		display: flex;
+		flex-direction: column;
+		margin-top: 1em;
+		width: 100%;
+
+		&__player,
+		&__computer {
+			display: flex;
+			justify-content: space-between;
+			align-items: center;
+			margin-top: 1em;
+			padding-inline: 3em;
+
+			
```

Comment:
Puste linie do wywalenia, zwykle po zamknieciu deklaracji `}` dajemy też pustą linie (zwykle to robisz)